client/cliClient.py

```diff
@@ -48,14 +48,36 @@ def _add_response(response: Packet) -> str:
     code = ResponseCode(data['responseCode'])
     if code == ResponseCode.OK:
         return "Successfully added you to the schedule!"
+    elif code == ResponseCode.CONFLICT:
+        return "Can't work on the same day as " + ', '.join(data['results'])
+    elif code == ResponseCode.UNEXPECTED:
```

Comment:
Is this because we couldn't find their uid anywhere? If not we need a response for that.

Reply:
uisng 404